src/engines/kotor/gui/chargen/chargeninfo.cpp

```diff
@@ -197,6 +197,48 @@ Creature *CharacterGenerationInfo::getCharacter() const {
 	return creature.release();
 }
 
+void CharacterGenerationInfo::recreateHead() {
```

Comment:
This is like the third or fourth copy of the same code, here in CharacterGenerationInfo and also in class Creature.

Can you abstract this away, so that there's one function that creates the model name for the head and one for the body, used by both class Creature and class CharacterGenerationInfo?